src/modal-create-component.js

```diff
@@ -226,12 +234,18 @@ function addPaths (evt) {
     views.pathAdd.textContent = 'Add root paths:'
     views.pathAdd.addEventListener('click', addPath.bind(this), false);
 
+    views.pathAddAuto = document.createElement('button');
+    views.pathAddAuto.classList.add('inline-block', 'btn', 'btn-warning', 'btn-xs', 'icon', 'icon-file-add');
+    views.pathAddAuto.textContent = 'Add tree view contents'
```

Comment:
Also, it could make more sense to make the first buttons like, `search for root folder`. What are your thoughts on this @DamnedScholar?